docs/devenv_rhel8.md

```diff
@@ -83,7 +83,6 @@ The artifact of the build is the `microshift` executable file located in the `_o
 ### RPM Packages
 Run make command with the `rpm` or `srpm` argument in the top-level directory. 
 ```bash
-make clean
```

Comment:
We need clean because these instructions are reused in the automation script and it should be idempotent. If clean is not present, the RPM install command may fail because it'll pick up RPMs from more than one build.

Reply:
I was following the instructions in the doc manually (as i believe most of the folks trying to setup their local environment). In the `Executable` section above, make builds the microshift executable, but `make clean` in this section deletes that, and you need to make it again. I think having a specific make target for cleaning executable and RPMS would be better i think. what do you think?

Reply:
I would not add a separate target just for deleting microshift executable. Note that RPM build rebuilds its own executable in a different location.
What about moving the 'make clean' before 'make' in the previous section?

Reply:
yeah, that's reasonable. Moved make clean to Executable section.